packages/functional-tests/tests/subscription-tests/support.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '../../lib/fixtures/standard';
+
+test.describe('support form without valid session', () => {
+  test.beforeEach(() => {
+    test.slow();
```

Comment:
There is only one test case, could you put the slow in their? Also any particular reason why it might be slow?